staging/src/k8s.io/code-generator/cmd/informer-gen/generators/factoryinterface.go

```diff
@@ -76,6 +76,7 @@ func (g *factoryInterfaceGenerator) GenerateType(c *generator.Context, t *types.
 }
 
 var externalSharedInformerFactoryInterface = `
+// NewInformerFunc is a function returning a SharedIndexInformer
```

Comment:
Nit: probably not worth changing, but next time think about customizing the comment for the type. (and ending with a period ;) )

Reply:
codegen has to be updated for tests to pass, so I'll solve the nit as well. Should I rebase or add them as new commits?